$MirOS: ports/devel/omniORB/patches/patch-configure_ac,v 1.1 2008/08/13 10:02:02 tg Exp $
--- configure.ac.orig	Tue Jul 15 11:03:57 2008
+++ configure.ac	Sun Nov  2 03:26:04 2008
@@ -182,6 +182,8 @@ case "$host" in
   *-*-freebsd6*) plat_name="FreeBSD";  plat_def="__freebsd__";  os_v="6";;
   *-*-freebsd7*) plat_name="FreeBSD";  plat_def="__freebsd__";  os_v="7";;
   *-*-kfreebsd*) plat_name="kFreeBSD";  plat_def="__FreeBSD_kernel__";  os_v="6";;
+  *-*-midnightbsd*) plat_name="MidnightBSD"; plat_def="__midnightbsd__"; os_v="0";;
+  *-*-mirbsd*)   plat_name="MirBSD";   plat_def="__mirbsd__";   os_v="10";;
   *-*-netbsd*)   plat_name="NetBSD";   plat_def="__netbsd__";   os_v="1";;
   *-*-openbsd*)  plat_name="OpenBSD";  plat_def="__openbsd__";  os_v="3";;
   *-*-sco*)      plat_name="OSR5";     plat_def="__osr5__";     os_v="5";;
